src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -109527,5 +109527,764 @@ class Program
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics();
         }
+
+        [Fact, WorkItem(35816, "https://github.com/dotnet/roslyn/issues/35816")]
```

Comment:
>Fact [](start = 9, length = 4)

I'll review tomorrow, but one quick note: please update the speclet: [https://github.com/dotnet/roslyn/blob/master/docs/features/nullable-reference-types.md#null-tests](https://github.com/dotnet/roslyn/blob/master/docs/features/nullable-reference-types.md#null-tests)